quick-integration.md

```diff
@@ -40,7 +52,7 @@ Once a transaction is funded you can use our webhook facilities to listen in cha
 
```

Comment:
We could add some notes here and/or in the webhook guide that links to the SDKs as well that shows how to use them